### PR TITLE
Add blog post on collaboration of Alasca and SCS

### PR DESCRIPTION
**Pull request by itrich**

### CODE REVIEW

**Comment by itrich** [comment]
Please don't merge this before #500 as the linked PR contains a minor fix for blog postings.

**Review by maxwolfs — APPROVED** [review]
lgtm!

File: _i18n/de/_posts/blog/2023-01-16-collaboration-of-alasca-and-scs.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+author:
+  - "Kurt Garloff"
+  - "Marius Feldmann"
+avatar:
+  - "kgarloff.jpg"
+  - "Marius_Feldmann-200x200.webp"
+image: "blog/fistbump.jpg"
+---
+
+The Sovereign Cloud Stack (SCS) project is a platform for open collaboration where a provider-focused ecosystem collaborates to create and operate cloud and container technology together in order  to support their customers to achieve higher levels of Digital Sovereignty. Common certifiable standards, a fully open modular reference implementation and transparency and knowledge sharing for operational processes are main outcomes from this collaboration. SCS is hosted by the OSB Alliance e.V. and funded by the BMWK (German Ministry for Economic Affairs and Climate Action).
```
Comment (garloff) [inline _i18n/de/_posts/blog/2023-01-16-collaboration-of-alasca-and-scs.md]: SCS is a project by the OSB Alliance e.V. and funded by the BMWK.

Comment (garloff) [reply _i18n/de/_posts/blog/2023-01-16-collaboration-of-alasca-and-scs.md]: Actually, we have a german translation.
So I'll push updated texts into this PR.

**Review by garloff — APPROVED** [review]
/lgtm

**Pull request merged**